features/product/viewing_products/not_seeing_lowest_price_for_product_that_has_all_taxons_excluded_on_channel.feature

```diff
@@ -0,0 +1,50 @@
+@viewing_products
+Feature: Not seeing the lowest price for a product that has all taxons excluded on the channel
+    In order to show the product's lowest price only in applicable taxons
+    As a Guest
+    I don't want to see the product's lowest price in taxons that have been excluded on the channel
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store classifies its products as "Category"
+        And the "Category" taxon has children taxons "Clothes" and "PHP"
+        And the "Clothes" taxon has child taxon "T-Shirts"
+        And the store has a product "T-Shirt Banana" priced at "$21.00"
+        And it belongs to "T-Shirts"
+        And this product's price changed to "$10.00" and original price changed to "$21.00"
+        And the store also has a product "T-Shirt Watermelon" priced at "$22.00"
+        And it belongs to "T-Shirts"
+        And the store also has a product "T-Shirt PHP" priced at "$23.00"
+        And it belongs to "T-Shirts"
+        And it belongs to "PHP"
```

Comment:
```suggestion
        And the store also has a product "T-Shirt PHP" priced at "$23.00" that belongs to "T-Shirts" and "PHP"
```
OR maybe?
```suggestion
        And the store also has a product "T-Shirt PHP" priced at "$23.00"
        And it belongs to "T-Shirts" and "PHP"
```

Reply:
Yup, you are right, I agree that this would be better, but when I could I based it on the steps that are defined in Sylius/Sylius and I did not want to add a whole new context just for this one step